R/dp_make_params.R

```diff
@@ -0,0 +1,126 @@
+#' @title Make dp params to connect to the Data Product Board
+#' @description Make dp params to connect to the Data Product Board. This is needed
+#' prior to interacting with the content of the board.
+#' @param url Repository URL
+#' @param repo_token Personal Access Token (PAT) needed to access the contents of
+#' the private repository
+#' @return Named list of dp params
+#'
+#' @examples
+#' \dontrun{
+#' url <- "https://github.com/<USERNAME>/<REPOSITORY NAME>"
+#' dp_params <- dp_make_params(url = url, repo_token = Sys.getenv("GITHUB_PAT"))
+#' @export
+
+dp_make_params <- function(url, repo_token){
```

Comment:
What do others think of having this function look for an existing GITHUB_PAT variable as the default? Then you would only have to provide the repo_token arg if you want to override GITHUB_PAT, or don't have it set at all.